websites/S/Simkl/presence.ts

```diff
@@ -0,0 +1,374 @@
+const presence = new Presence({
+		clientId: "1290874564582051855",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+function textContent(tags: string) {
+	return document.querySelector(tags)?.textContent?.trim();
+}
+
+function getImage(tags: string) {
+	return document.querySelector<HTMLImageElement>(tags)?.src;
+}
+
+presence.on("UpdateData", async () => {
+	const { pathname, search } = document.location,
+		path = pathname.split("/"),
+		presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+		};
+
+	switch (path[1]) {
+		case "tv": {
+			presenceData.details = "Exploring TV Shows";
+			if (path[2]) {
+				switch (path[2]) {
+					case "calendar": {
+						presenceData.details = "Viewing TV Shows Calendar";
+						break;
+					}
+					case "discover": {
+						presenceData.details = "Discovering TV Shows";
+						break;
+					}
+					case "today": {
+						presenceData.details = "Viewing Today's TV Shows Calendar";
+						break;
+					}
+					case "movies": {
+						presenceData.details = "Viewing Animes Movies";
+						break;
+					}
+					case "best-shows": {
+						presenceData.details = "Viewing Best TV Shows";
+						break;
+					}
+					case "premieres": {
+						presenceData.details = "Viewing Premieres TV Shows";
+						break;
+					}
+				}
+				if (path[3]) {
+					presenceData.details = textContent("h1.headDetail");
+					presenceData.largeImageKey = getImage("#detailPosterImg");
+					if (path[4] && path[4].startsWith("countdown")) {
+						presenceData.details = textContent(".heading").replace(
+							"Countdown",
+							""
+						);
+						presenceData.state = textContent(".simkltvcountdownsubtitle");
+						presenceData.largeImageKey = getImage(
+							".simkltvcountdowninfoleftiage"
+						);
+					} else if (path[4] && path[4].startsWith("review")) {
+						presenceData.details = `Reading a ${textContent(
+							"#global_div > div.SimklHeaderBgShaddow > div > div > div.SimklTVReviewHeaderBottom > table > tbody > tr > td > table > tbody > tr > td:nth-child(2) > table > tbody > tr:nth-child(1) > td"
+						)}`;
+						presenceData.state = textContent(
+							"#global_div > div.SimklHeaderBgShaddow > div > div > div.SimklTVReviewHeaderBottom > table > tbody > tr > td > table > tbody > tr > td:nth-child(2) > table > tbody > tr:nth-child(2) > td"
```

Comment:
try to shorten selectors where possible.
```suggestion
							"span.ajLinkInside"
```